analyses/focal-cn-file-preparation/03-prepare-cn-file.R

```diff
@@ -0,0 +1,306 @@
+# This script converts a seg file into a tsv file with CN information and gene
+# annotation.
+#
+# Code adapted from the PPTC PDX Focal Copy NUmber and SV Plots repository here:
+# https://github.com/marislab/pptc-pdx-copy-number-and-SVs/blob/master/R/focal-CN-revision.R
+#
+# Chante Bethell for CCDL 2019 and Jo Lynne Rokita
+#
+# #### USAGE
+# This script is intended to be run via the command line from the top directory
+# of the repository as follows:
+#
+# Rscript 'analyses/oncoprint-landscape/01-prepare-cn-file.R'
+
```

Comment:
Unless this script would work without specifying anything,  I suspect this usage example is incomplete?